src/sentry/models/event.py

```diff
@@ -476,13 +476,57 @@ def id(self):
         # have to reference the row id anyway.
         return self.event_id
 
-    @property
-    def next_event(self):
-        return None
+    def next_event_id(self, environments=[]):
+        from sentry.utils import snuba
 
-    @property
-    def prev_event(self):
-        return None
+        conditions = [['event_id', '!=', self.id]]
```

Comment:
I think you still need the `timestamp > self.timestamp OR (timestamp == self.timestamp AND event_id > self.id)` type condition in here.

Reply:
How would you do a string comparison function for event_id like that in a Snuba condition?

Reply:
you can use `>`, `>=` etc on strings, 

The tough part I think is that snuba nested conditions are strung together as an `AND` of `OR`s, and you might need them nested the other way, ie https://github.com/getsentry/snuba#conditions

Reply:
`conditions = [["timestamp", ">=", self.timestamp], [["timestamp", ">", self.timestamp], ["id", ">", self.id]]]` is the above query rearranged to be an AND of ORs. 

and you can definitely do inequality operators on Strings and FixedStrings in clickhouse, so I don't think the comparison should be a problem

Reply:
Thanks, updated 17ed3e1